AxonDeepSeg/segment.py

```diff
@@ -7,398 +7,403 @@
+                                                                                'where the image(s) to segment is/are located.')
+
+    ap.add_argument("-m", "--model", required=False, help='Folder where the model is located. \n'+
+                                                          'The default SEM model path is: \n'+str(default_SEM_path)+'\n'+
+                                                          'The default TEM model path is: \n'+str(default_TEM_path)+'\n')
+    ap.add_argument('-s', '--sizepixel', required=False, help='Pixel size of the image(s) to segment, in micrometers. \n'+
+                                                              'If no pixel size is specified, a pixel_size_in_micrometer.txt \n'+
+                                                              'file needs to be added to the image folder path. The pixel size \n'+
+                                                              'in that file will be used for the segmentation.',
+                                                              default=None)
+    ap.add_argument('-v', '--verbose', required=False, type=int, choices=list(range(0,4)), help='Verbosity level. \n'+
+                                                            '0 (default) : Displays the progress bar for the segmentation. \n'+
+                                                            '1: Also displays the path of the image(s) being segmented. \n'+
+                                                            '2: Also displays the information about the prediction step \n'+
+                                                            '   for the segmentation of current sample. \n'+
+                                                            '3: Also displays the patch number being processed in the current sample.',
+                                                            default=0)
+    ap.add_argument('-o', '--overlap', required=False, type=int, help='Overlap value (in pixels) of the patches when doing the segmentation. \n'+
+                                                            'Higher values of overlap can improve the segmentation at patch borders, \n'+
+                                                            'but also increase the segmentation time. \n'+
+                                                            'Default value: '+str(default_overlap)+'\n'+
+                                                            'Recommended range of values: [10-100]. \n',
+                                                            default=25)
+    ap._action_groups.reverse()
+
+    # Processing the arguments
+    args = vars(ap.parse_args(argv))
+    type_ = str(args["type"])
+    verbosity_level = int(args["verbose"])
+    overlap_value = int(args["overlap"])
+    if args["sizepixel"] is not None:
+        psm = float(args["sizepixel"])
+    else:
+        psm = None
+    path_target_list = [Path(path) for path in args["imgpath"]]
+    new_path = args["model"]
```

Comment:
Why did `path_target_list = args["imgpath"]` have to be converted to `path_target_list = [Path(path) for path in args["imgpath"]]`, but `new_path = args["model"]` doesn't need a similar conversion?

Reply:
@maf88 You marked this as resolved without answering my question. If you want to keep it this way that's fine, I was just trying to understand this a bit deeper.